silk/hw/hw_module.py

```diff
@@ -114,6 +117,17 @@ def get_otns_vis_position(self):
     def get_otns_vis_node_id(self):
         return self._node_id
 
+    def get_otns_vis_layout_center(self):
+        parts = self._layout_center.split(",")
```

Comment:
We should parse the string once in `__init__`, rather than parsing it every time when `get_otns_vis_layout_center` is called.